frontend/lib/resolution/resolution-queries.cpp

```diff
@@ -2252,6 +2261,14 @@ filterCandidatesInitial(Context* context,
 
   return QUERY_END(result);
 }
+static const std::vector<const TypedFnSignature*>&
+filterCandidatesInitialWrapper(Context* context,
+                               std::vector<BorrowedIdsWithName>&& lst,
+                               const CallInfo& call) {
+  auto lstMove = std::move(lst);
+  auto callCopy = call;
+  return filterCandidatesInitial(context, lstMove, callCopy);
```

Comment:
This will copy `lstMove` again unless it's moved again here. I think you can rewrite this as:
```C++
static const std::vector<const TypedFnSignature*>&
filterCandidatesInitialWrapper(Context* context,
                               std::vector<BorrowedIdsWithName>&& lst,
                               const CallInfo& call) {
  return filterCandidatesInitial(context, std::move(lst), call);
}
```